pages/articleDetails/[id]/index.js

```diff
@@ -0,0 +1,87 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import { fetcher } from "../../../helpers/api";
+import Image from "next/image";
+import styled from "styled-components";
+import css from "styled-jsx/css";
+import Svg from "../../../components/Svg";
+import Link from "next/link";
+
+function articleDetails() {
+  const { query } = useRouter();
+  const { id } = query;
+  const { data: article, error } = useSWR(`/api/articles/${id}`, fetcher);
+
+  if (!article) return <p>Artikel Details werden geladen...</p>;
+  if (error) return <p>Fehler! keine Artikel Details gefunden.</p>;
```

Comment:
Just thinking... Would it be better switching these two lines? Right now: If an error occurs, the article is not there, just line 15 is rendered and line 16 never gets rendered. But I might be wrong. But, thinking switching these two lines would be better if an error orrcurs.

```suggestion
  if (error) return <p>Fehler! keine Artikel Details gefunden.</p>;
  if (!article) return <p>Artikel Details werden geladen...</p>;
```